diagnostic.md

```diff
@@ -150,7 +167,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+All divs except divFour would be on the top row next to each other, with divOne all the way on the right. divFour would be in the second row, stretched out across the screen.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5
```